### PR TITLE
Add share SVG icons

### PR DESCRIPTION
**Pull request by liywjl**
## What is the purpose of this change?

Expands list of supported SVG icons to include social apps such as:

- whatsapp
- linkedin
- messanger
- pinterest

## What does this change?

- Add SVGs
- Export SVGs

## Screenshots

Screenshot 2021-01-20 at 12 06 39
Screenshot 2021-01-20 at 12 06 37
Screenshot 2021-01-20 at 12 06 33
Screenshot 2021-01-20 at 12 06 28

### Accessibility

-   [ ] [Tested with screen reader](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/78ac51)
-   [ ] [Navigable with keyboard](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/009027)
-   [ ] [Colour contrast passed](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/58dbf2)
-   [ ] [The component doesn't use only colour to convey meaning](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/61524d)

### Cross browser and device testing

-   [ ] Tested with touch screen device

### Responsiveness

-   [ ] Tested at all breakpoints
-   [ ] Tested with with long text
-   [ ] Stretched to fill a wide container

### Documentation

-   [ ] Full API surface area is documented in the README
-   [x] Examples in Storybook

### CODE REVIEW

File: src/core/icons/linkedin.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+export const SvgLinkedin = (): JSX.Element => (
+	<svg
+		width="30"
+		height="27"
+		viewBox="0 0 30 27"
```
Comment (SiAdcock) [inline src/core/icons/linkedin.tsx:7]: Oof, the viewbox is sized incorrectly, it should be 30x30. We'll fix that in Figma, sorry!

File: src/core/icons/index.ts [hunk 2]
```diff
@@ -24,15 +24,19 @@ export * from "./facebook"
 export * from "./house"
 export * from "./indent"
 export * from "./info"
+export * from "./linkedin"
+export * from "./messanger"
```
Comment (SiAdcock) [inline src/core/icons/index.ts:28]: Typo in file name: `messenger`

File: src/core/icons/messanger.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+export const SvgMessenger = (): JSX.Element => (
+	<svg width="66" height="66" viewBox="-12 -12 90 90">
```
Comment (SiAdcock) [inline src/core/icons/messanger.tsx:4]: The viewport for this icon is incorrect. Could you wait until this icon is added to Figma?

File: src/core/icons/pinterest.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+export const SvgPinterest = (): JSX.Element => (
+	<svg
+		width="30"
+		height="30"
+		viewBox="0 0 30 30"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
```
Comment (SiAdcock) [inline src/core/icons/pinterest.tsx:9]: Could you remove the `width`, `height` and `fill` attributes? These should be set in CSS

File: src/core/icons/whatsapp.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+export const SvgWhatsapp = (): JSX.Element => (
+	<svg
+		width="30px"
+		height="30px"
+		viewBox="0 0 30 30"
+		enableBackground="new 0 0 30 30"
```
Comment (SiAdcock) [inline src/core/icons/whatsapp.tsx:8]: Is `enableBackground` doing anything useful?

**Review by SiAdcock — APPROVED** [review]
👍

**Comment by liywjl** [comment]
I cannot approve, so instead I give 👍

**Pull request merged**